frontend/src/utils.ts

```diff
@@ -13,3 +13,7 @@ function formatDate(date: DateString) {
    });
 }
 
+export function toDateString(date: Date): DateString {
```

Comment:
We may want to move over some of the util functions from `pulldasher/lib/utils.js` as there seems to be more functions that deal with dates in there as well https://github.com/iFixit/pulldasher/blob/923c28d9a8a1c13d3d5e8abca03d2058fd96b30d/lib/utils.js#L34-L36

Reply:
Yes, there are more date functions we should copy them as needed. At the moment, the backend doesn't support typescript, so we can't import them directly.